bors: only check Essential CI before merging

otan opened the pull request:
We don't need testrace in bors, so avoid that check. This should improve
bors speeds when many packages have changed.

Release note: None

Release justification: non-production code change

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/53370)
<!-- Reviewable:end -->

jlinder reviewed (approved):
I think this may also require configuring Essential CI to post status updates back to GitHub.

otan commented:
hmm, need a github access token for that.

otan commented:
looks suitably quiet!

bors r=jlinder

otan committed:
bors: only check Essential CI before merging

We don't need testrace in bors, so avoid that check. This should improve
bors speeds when many packages have changed.

Release justification: non-production code changes

Release note: None

otan commented:
bors r=jlinder

otan commented:
bors r=jlinder p=1

this has to be the only PR in the queue, i think.

otan commented:
bors r+
